proto/ecosystem-incentive/params.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+package ununifi.ecosystemincentive;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/UnUniFi/chain/x/ecosystem-incentive/types";
+
+// Params defines the parameters for the module.
+message Params {
+  repeated RewardParams reward_params = 1 [ (gogoproto.moretags) = "yaml:\"reward_params\"" ];
+  uint64 MaxIncentiveUnitIdLen = 2 [ (gogoproto.moretags) = "yaml:\"max_incentive_unit_id\"" ];
```

Comment:
can you make it to max_incentive_unit_id_len? as well as tag from max_incentive_unit_id => max_incentive_unit_id_len

Reply:
modified it.